tests/stdlib/thashes.nim

```diff
@@ -75,3 +75,15 @@ block largeSize: # longer than 4 characters
   doAssert hash(xx) == hash(ssl, 0, 4)
   doAssert hash(xx, 0, 3) == hash(xxl, 0, 3)
   doAssert hash(xx, 0, 3) == hash(ssl, 0, 3)
+
+proc main() =
```

Comment:
for future work:
unfortunately wrapping the other blocks inside main fails at CT:
`Error: cannot generate code for: mSlice`

Reply:
I tried them too :(

Reply:
```nim
import hashes

proc main() =
  echo hash(cstring"abracadabra")

static: main()
```

Reply:
Even for string types

It doesn't print 
```nim
import hashes

proc main() =
  echo hash("abracadabra")

static: main()
main()
```

Reply:
that's https://github.com/nim-lang/Nim/issues/16061 I think

Reply:
After fixing that, RT still prints wrong result:
```nim
import hashes

proc main() =
  echo hash(cstring"abracadabra")

static: main()
main()
```
both in JS/C VM vs gen
97309975
0

Reply:
I will fix them(may be after this PR)